git-try-push/main.js

```diff
@@ -45,15 +46,19 @@ async function main() {
         // Loop specified number of tries.
         for (let i = 0; i < tries; i++) {
             try {
+                // We can use `${branch}:${remote_branch}` as a catch-all. But we
+                // want to simplify the log output a bit because `branch` and
+                // `remote_branch` are often the same.
+                const refspec = (branch == remote_branch) ? branch : `${branch}:${remote_branch}`
```

Comment:
Thinking about this some more: maybe we can/should use `origin_branch` here instead of using a new input?

Reply:
In that case, we'll essentially change the definition of `origin_branch` to "Remote branch to rebase on *and push to*". They are not necessarily the same, but hmm, I can't really come up with a real-world use case where a branch to rebase on needs to be different from the one to push to. So 👍